plugin/adsk/scripts/mayaUsd_layerEditorFileDialogs.mel

```diff
@@ -117,8 +117,20 @@ global proc string UsdLayerEditor_SaveLayerFileDialog(int $isRootLayer, string $
         $selectionCallback = "mayaUsd_USDSubLayerFileRelative_SelectionChanged";
     }
 
+    string $titleCaption;
+    string $actionCaption;
+    if ($captionMode == 0) {
+        $titleCaption = "Save As";
+        $actionCaption = "Save";
+    } else {
+        $titleCaption = "Set As";
+        $actionCaption = "Set";
```

Comment:
These strings should really be in resource dictionary for future translation. But actually these strings are probably already in one (or more) of the Maya dictionary files. Search for uses of `uiRes` and you can see how we can directly use a string from Maya.

Reply:
@seando-adsk If it's ok with you, I would have this PR merged as is and apply this change with my next PR, as there will be more changes going into Batch Save UI.